nginx/conf.d/redirects/redirects.conf

```diff
@@ -806,3 +806,5 @@ rewrite ^/co-occurring-disorders/season-affective-disorder/?$ https://$host/co-o
 rewrite ^/co-occurring-disorders/self-medicating/?$ https://$host/co-occurring-disorders/statistics/ permanent;
 #FACILITY-8514
 rewrite ^/addiction-treatment/levels-of-care/inpatient-vs-outpatient/?$ https://$host/las-vegas-program/levels-of-care/inpatient-vs-outpatient/ permanent;
+#FACILITY-8928
+rewrite ^/co-occurring-disorders/impulse-control-disorder/?$ https://$host/co-occurring-disorders/ permanent;
```

Comment:
this redirect has already been added on master per this task https://recoverybrands.atlassian.net/browse/FACILITY-7295